testing/KeyboardReport.cpp

```diff
@@ -34,8 +34,19 @@ std::vector<uint8_t> KeyboardReport::ActiveKeycodes() const {
     uint8_t key_code = report_data_.keys[i / 8] & bit;
     if (key_code) active_keys.push_back(i);
   }
+  for (uint8_t i{0}; i < 8; ++i) {
+    uint8_t bit = 1 << i;
```

Comment:
Very minor: we've got a gcc platform level  _BV(i) macro for this instead of the bitshift.

Reply:
To me, that's just yet another mysterious macro that I need to go searching for, with no real clues about its provenance. I'd really rather not use it.